### PR TITLE
Create helm boilerplate

### PR DESCRIPTION
**Pull request by WillNilges**

### CODE REVIEW

File: .github/workflows/deploy-to-k8s.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        sudo wg set wg1 listen-port 48123 private-key privatekey peer ${{ secrets.WIREGUARD_PEER_PUBLIC_KEY }} allowed-ips 0.0.0.0/0 endpoint ${{ secrets.WIREGUARD_ENDPOINT }}
+        sudo ip link set up dev wg1
+
+    - name: Deploy Helm Chart
+      run: |
+        # Get the kubeconfig
+        scp ${{ secrets.SSH_USER }}@${{ secrets.SSH_TARGET_IP }}:~/.kube/config ./
+
+        # Create the namespace if necessary
+        kubectl --kubeconfig ./config --server https://${{ secrets.SSH_TARGET_IP }}:6443 create namespace ${{ vars.APP_NAMESPACE }} || echo namespace already exists
```
Comment (james-otten) [inline .github/workflows/deploy-to-k8s.yaml:43]: I don't think you need this line

File: infra/helm/meshforms/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+apiVersion: v2
+name: meshforms
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
```
Comment (james-otten) [inline infra/helm/meshforms/Chart.yaml:5]: I think you wanted all this removed in meshdb

File: infra/helm/meshforms/templates/NOTES.txt [hunk 3]
```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```
Comment (james-otten) [inline infra/helm/meshforms/templates/NOTES.txt:1]: Is this used?

File: infra/helm/meshforms/templates/secrets.yaml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: meshforms-secrets
+  namespace: {{ .Values.meshforms_app_namespace }}
```
Comment (james-otten) [inline infra/helm/meshforms/templates/secrets.yaml:5]: Looks like you are handing this via helm, may want to remove for consistency

File: infra/helm/meshforms/values.yaml [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    - host: chart-example.local
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+resources: {}
```
[the remark this answers is not in the document]
Comment (james-otten) [reply infra/helm/meshforms/values.yaml:42]: Send that email

**Review by james-otten — APPROVED** [review]
Very nice!

File: infra/helm/meshforms/Chart.yaml (hunk 2 above)
Comment (WillNilges) [reply infra/helm/meshforms/Chart.yaml:5]: Actually, these variables are necessary and we should keep them. I think they have something to do with stuff like `.Chart.Name` working.

File: infra/helm/meshforms/templates/NOTES.txt (hunk 3 above)
Comment (WillNilges) [reply infra/helm/meshforms/templates/NOTES.txt:1]: I guess this is part of the template

File: infra/helm/meshforms/templates/secrets.yaml (hunk 4 above)
Comment (WillNilges) [reply infra/helm/meshforms/templates/secrets.yaml:5]: What do you mean by "this?" The secrets file? I use this within `deployment.yaml`, though it's not very templated and I need to fix that :/

File: infra/helm/meshforms/Chart.yaml (hunk 2 above)
Comment (james-otten) [reply infra/helm/meshforms/Chart.yaml:5]: Comments

File: infra/helm/meshforms/templates/secrets.yaml (hunk 4 above)
Comment (james-otten) [reply infra/helm/meshforms/templates/secrets.yaml:5]: Namespace

File: infra/helm/meshforms/Chart.yaml (hunk 2 above)
Comment (WillNilges) [reply infra/helm/meshforms/Chart.yaml:5]: oh god I need to go to bed.

File: infra/helm/meshforms/templates/NOTES.txt (hunk 3 above)
Comment (WillNilges) [reply infra/helm/meshforms/templates/NOTES.txt:1]: NOTES.txt is printed at the end of `helm install`? We don't really need it for internal charts, I suppose. 

https://helm.sh/docs/chart_template_guide/notes_files/

**Pull request merged**